spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskSchedulerController.java

```diff
@@ -119,7 +120,6 @@ public void save(@RequestParam("scheduleName") String scheduleName,
 			@RequestParam String properties,
 			@RequestParam(required = false) List<String> arguments) {
 		Map<String, String> propertiesToUse = DeploymentPropertiesUtils.parse(properties);
-		DeploymentPropertiesUtils.validateDeploymentProperties(propertiesToUse);
```

Comment:
I believe we would still want to validate the deployment properties (may be specific to scheduler properties to validate against their prefix) to make sure they only allow scheduler specific properties.

Reply:
Updated